pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -67,6 +67,9 @@ type CiliumNetworkPolicy struct {
 type CiliumNetworkPolicyStatus struct {
 	// Nodes is the Cilium policy status for each node
 	Nodes map[string]CiliumNetworkPolicyNodeStatus `json:"nodes,omitempty"`
+
+	// ChildrenPolicies is the Cilium Children policies status
+	ChildrenPolicies map[string]CiliumChildPolicyStatus `json:"childrenPolicies,omitempty"`
```

Comment:
Let's call this `Derivatives` to make it a bit clearer that we are referring to policy rules that are derived from this rule. Please extend the comment to specify what the map key describes. If it is the name of the policy, why does it have to be a map?